risc0/zkvm/src/exec/env.rs

```diff
@@ -131,18 +155,56 @@ impl<'a> ExecutorEnvBuilder<'a> {
     }
 
     /// Set a session limit, specified in number of cycles.
+    /// # Example
+    /// ```
+    /// # use risc0_zkvm::{
+    /// #   ExecutorEnv,
+    /// #   ExecutorEnvBuilder};
+    ///
+    /// const NEW_SESSION_LIMIT: usize = 32 * 1024 * 1024; // 32M cycles
+    ///
+    /// let env = ExecutorEnv::builder()
+    ///     .session_limit(NEW_SESSION_LIMIT)
+    ///     .build();
+    /// ```
     pub fn session_limit(&mut self, limit: usize) -> &mut Self {
         self.inner.session_limit = limit;
         self
     }
 
     /// Add environment variables to the guest environment.
+    /// # Example
+    /// ```
+    /// # use risc0_zkvm::{
+    /// #   ExecutorEnv,
+    /// #   ExecutorEnvBuilder};
+    /// # use std::collections::HashMap;
+    ///
+    /// let mut vars = HashMap::new();
+    /// vars.insert("VAR1".to_string(), "SOME_VALUE".to_string());
+    /// vars.insert("VAR2".to_string(), "SOME_VALUE".to_string());
+    ///
+    /// let env = ExecutorEnv::builder()
+    /// .env_vars(vars)
+    /// .build();
+    /// ```
     pub fn env_vars(&mut self, vars: HashMap<String, String>) -> &mut Self {
         self.inner.env_vars = vars;
         self
     }
 
     /// Add an environment variable to the guest environment.
+    /// # Example
+    /// ```
+    /// # use risc0_zkvm::{
+    /// #   ExecutorEnv,
+    /// #   ExecutorEnvBuilder};
+    /// # use std::collections::HashMap;
```

Comment:
I don't think this is needed

Reply:
Oops! Copied and pasted without pulling that out.